spring-cloud-function-samples/function-sample-task/build.gradle

```diff
@@ -46,6 +46,5 @@ dependencyManagement {
 
 dependencies {
 	compile('org.springframework.cloud:spring-cloud-function-task')
-	compile('org.springframework.cloud:spring-cloud-function-compiler')
```

Comment:
So this sample does use the compiler as it has `spring.cloud.function.compile` properties in its `application.yml`. However, this sample module is not specified in the base pom and therefore not included in the build commands (effectively disabled/inactive/etc..).

I think we either remove this sample or we re-work it so that its supplier/function/consumer do not come from dynamically compiled expressions. 

@olegz wdyt?

Reply:
I say we remove it. We have plenty of samples now